quarkchain/cluster/master.py

```diff
@@ -1293,6 +1293,7 @@ def handle_new_root_block_header(self, header, peer):
 
         for grpc_client in self.grpc_slave_pool:
             grpc_client.set_rootchain_confirmed_block()
+            grpc_client.add_root_block(r_block)
```

Comment:
can remove lien 1295

Reply:
Ok, then where should I add this method?

Reply:
no need to. it's not being used right now

Reply:
ok I got it.

Reply:
now we need to think about failure cases: what if the result doesn't succeed? should we handle failures differently for network failures (e.g. timeout) and application failures (e.g. the slave thinks the root block is malformed)? is Boolean type enough for this RPC? what do you think?

Reply:
Do we need to add more failure types in GrpcClient.add_root_block()? Maybe we can define a GrpcResponse struct in rpc.py and use different error code representing different failure types.
For the root block sent to the slave, do we need to first validate the content of it before we send it to the slave?